CHANGELOG.rst

```diff
@@ -12,6 +14,13 @@
   This functionality has been replaced by the ``exclude_positions`` argument
   to the ``infer`` and ``generate_ancestors`` functions.
 
+**Bugfixes**:
+
+- Individuals and populations in the sampledata file are kept in the returned tree
+  sequence, even if they are not referenced by any sample. The individual and population
+  ids should therefore stay the same between the sample data file and the inferred tree
```

Comment:
"Should" seems weak - what about "ids therefore guaranteed to the same between"

Reply:
Sure. Changed.